www/editclub

```diff
@@ -478,16 +478,15 @@ formCol("membersPublic", $membersPublic, "Membership",
         . "to everyone. If not, it will be visible only to club members.");
 
 
-echo "<tr><td></td><td><p><input type=image alt='Save Changes' "
-   . "value='Save Changes' name='save' "
-   . "src='img/blank.gif' class='save-button'> &nbsp; "
+echo "<tr><td></td><td><p><button value='Save Changes' name='save' "
+   . "class='fancy-button'>Save Changes</button> &nbsp; "
 
-   . "<input type=image alt='Reset Form (Discard Changes) "
+   . "<button class='fancy-button' "
    . "value='Reset Form (Discard Changes)' name='discard' "
```

Comment:
Is this `value='Reset Form (Discard Changes)'` used by anything? Can it be removed? (Also relevant to other changes in this MR)

I thought it's only relevant when submitting form data, but `name` is already used to identify the button. Is it exposed to screen readers?

Reply:
I didn't know, and since I was making a bunch of changes, I thought it might be safer to just leave it in. Seems harmless?